objects/Ranomizer/Step_0.gml

```diff
@@ -10,111 +10,13 @@ if (alarm_1_trigger = 1)
 	alarm[1] = 500;
 }
 
-if (crazy_level = 0)	//============================== Level 0
-{
-	overhead_lights_right.image_speed = 0;
-	overhead_lights_left.image_speed = 0;
-	oAudience.frequency = 0
-	oAudience.amplitude = 0
-}
-
-if (crazy_level = 1)	//============================== Level 1
-{
-	overhead_lights_right.image_speed = 0;
-	overhead_lights_left.image_speed = 0;
-	oAudience.frequency = 0.05
-	oAudience.amplitude = 3
-	//Drum_left.hsp = 5;
-	//Drum_right.hsp = 5;
-}
-
-if (crazy_level = 2)	//============================== Level 2
-{
-	overhead_lights_right.image_speed = 0.2;
-	overhead_lights_left.image_speed = 0.2;
-	oAudience.frequency = 0.05
-	oAudience.amplitude = 4
-	//Drum_left.hsp = 6;
-	//Drum_right.hsp = 6;
-}
+image_speed_vals = [0, 0, 0.2, 0.2, 0.4, 0.4, 0.6, 0.6, 0.8, 0.8, 1]
```

Comment:
Can you just add a few spaces so this array lines up w/ the other ones? Makes it easier to quickly visually compare values